queries.md

```diff
@@ -0,0 +1,51 @@
+# Query documentation
+
+## Highlights
+
+### Description of query
+
+The highlights query is composed of two parts:
+
+( A ) Select **1 item** from History where:
+
+* the item has a bookmark
+* the date the bookmark was created (`bookmarkDateCreated`) is in the last 3 days
+* the bookmark has been visited no more than 3 times (`visit_count`)
+
+Items are ordered (descending) by the date the bookmark was created (`bookmarkDateCreated`), meaning that newer item will come first.
+
+( B ) Then, select up to **20 items** from History, where:
+
+* the item was NOT visited (`last_visit_date`) in the last 30 minutes
```

Comment:
maybe it would be clearer to say that the last visit has to be more than 30 mins ago